app/models/spree/user.rb

```diff
@@ -37,7 +37,7 @@ class User < ApplicationRecord
                             foreign_key: :owner_id, inverse_of: :owner
     has_many :customers
     has_many :credit_cards
-
+    has_many :report_rendering_options, class_name:"::ReportRenderingOptions",dependent: :destroy
```

Comment:
Good, but I think `class_name` isn't necessary, because `enterprises` doesn't need it. Did you try that?

Also it would be nice to keep the empty line below. This helps divide up sections in the model file and makes it easier to read.

i.e.
```suggestion
    has_many :report_rendering_options, dependent: :destroy

```

Reply:
Hi @dacook 

I already tried it and it didn't work.
```ruby
3.0.3 :002 > Spree::User.first.report_rendering_options
  Spree::User Load (2.9ms)  SELECT "spree_users".* FROM "spree_users" ORDER BY "spree_users"."id" ASC LIMIT $1  [["LIMIT", 1]]
/home/work/.rvm/gems/ruby-3.0.3/gems/activerecord-6.1.7/lib/active_record/inheritance.rb:237:in `compute_type': uninitialized constant Spree::User::ReportRenderingOption (NameError)
```